mods/persistence/controllers/subsystems/trade_beacons.dm

```diff
@@ -0,0 +1,26 @@
+SUBSYSTEM_DEF(trade_beacons)
+	name = "Trade Beacons"
+	wait = 5 MINUTES
+	priority = SS_PRIORITY_MONEY_ACCOUNTS
+	var/list/all_trade_beacons = list()
+	var/list/wanted_trade_beacons = list(/obj/effect/overmap/trade_beacon/test_beacon, /obj/effect/overmap/trade_beacon/test_beacon2) // list(//obj/effect/overmap/trade_beacon/example, /obj/effect/overmap/trade_beacon/steel, /obj/effect/overmap/trade_beacon/xandahar)
+	var/last_cycle = 0
+
+/datum/controller/subsystem/trade_beacons/Destroy()
+	QDEL_NULL_LIST(all_trade_beacons)
+	. = ..()
+
+/datum/controller/subsystem/trade_beacons/Initialize()
+	all_trade_beacons = list()
+	for(var/x in wanted_trade_beacons)
```

Comment:
You should add #ifndef UNIT_TEST before the for loop here and #endif after the for loop, like shown below. That should fix the unit test GC issue.
![image](https://user-images.githubusercontent.com/20494662/225182455-67bd583e-f65c-499a-8d0e-a20179305eeb.png)

Reply:
fixed, thanks.